charts/seed-terraformer/charts/azure-backup/templates/_main.tf

```diff
@@ -7,7 +7,7 @@ provider "azurerm" {
 }
 
 resource "azurerm_resource_group" "rg" {
-  name     = "{{ required "clusterName is required" .Values.clusterName }}-backup"
+  name     = "{{ required "azure.storageAccountName is required" .Values.azure.resourceGroupName }}"
```

Comment:
Copy/paste error: `azure.storageAccountName` should be `azure.resourceGroupName`